src/controllers/taskController.js

```diff
@@ -583,7 +583,7 @@ const taskController = function (Task) {
     // const isFine = true;
 
     // const dbTasks = [];
-    // console.log('fixedtask', fixedTasks.length);
+     console.log('fixedtask', fixedTasks);
```

Comment:
Please remove this change since it's not related to this PR.

Reply:
Done!